bundles/integration/admin-layerselector/templates/adminLayerSettingsTemplate.html

```diff
@@ -252,7 +252,7 @@
         </div>
     </div>
 
-    <!-- % if(model.getStyles().length > 1) { % -->
+    <% if (layerType !== "vectortile" && layerType !== "tiles3d") { %>
```

Comment:
Why is default style not selectable for these? I guess it doesn't make much sense to have the select, but it kinda bothers me to see layertype-specific stuff on the "generic" template.